TGC Comprehensive Rules.md

```diff
@@ -0,0 +1,183 @@
+        1.5.1.x: The opponent is not allowed to see the discarded cards but can know how many cards you chose to discard.  <br>
+    1.5.2: They can perform this up to 3 times. <br>
+    1.5.3: After both players have finished mulliganing shuffle all discarded cards back into the deck. <br>
+
+1.6: Both players starts with 20 Heathshards (or HS) and 1 Plasma. <br>
+1.7: The game begins on the Draw Phase of the player who won the coin toss. <br>
+1.8: The player that plays second keeps the coin and places it besides them, flipped on heads. <br>
+     1.9.x: This is relevant for the \"Flip Coin\" action, detailed in 3.6 <br>
+
+1.9: Turns in this game are divided in 5 parts, called the phases. <br>
+    1.9.a: Draw Phase <br>
+    1.9.b: Standby Phase <br>
+    1.9.c: Play Phase <br>
+    1.9.d: Battle Phase <br>
+    1.9.e: End Phase <br>
+        1.9.x: There is no \"Play Phase 2\". You do not get to return to a Play Phase after the Battle Phase. <br>
+
+1.10: During the Draw Phase a number of automatic game actions happen, in the following order. <br>
+    1.10.1: The turn player\'s mana gets set to the amount of Draw Phases they have had in total, including this one. <br>
+        1.10.1.x: So on their first turn they get 1 mana, on their second 2 mana, on their third 3 mana, etc. <br>
+    1.10.2: The turn player draws a card. <br>
+    1.10.3: The turn player untaps all their tapped cards. <br>
+    1.10.4: The "Pass Phase" action is immediately performed. <br>
+    1.10.5: This Pass Phase does not create an Open gamestate. <br>
+
+1.11: Terminology definitions: <br>
+    1.11.a: When you draw a card you add the top card of your deck to your hand. <br>
+    1.11.b: When a card is destroyed it is moved to the discard pile <br>
+        1.11.b.1: When a card is destroyed all cards attached to it also get destroyed. <br>
+    1.11.c: When a card is tapped it is flipped sideways. <br>
+    1.11.d: When a card is untapped it is returned to the upright position <br>
+
+1.12: Any player that have their HP reduced to, set to or at 0 loses the game automatically. <br>
+1.12.1: When this happens the other player immediately wins the game. <br>
+    1.12.x: This can happen even if the game is in a Closed Gamestate <br>
+
+<h2> Open and Closed States </h2>  <br>
+
+2: The gamestate can either be Open or Closed. <br>
+
+2.1: In a Closed gamestate no players may take any actions with the sole exception of: <br>
+    2.1.a: Surrendering <br>
+    2.1.b: When resolving an activated Effect that, as a part of its effect, allows its user to make a choice <br>
+
+2.2: In an Open gamestate then one of the players may take actions, depending on the type of Open gamestate, the phase and game information. <br>
+
+2.3: There are three types of Open gamestates: Open Play Gamestate, Open Battle Gamestate and the Open Quick Gamestate <br>
+    2.3.x: These can be refered to as Open Play, Open Battle and Open Quick. <br>
```

Comment:
To me this section can be truncated/removed as long as specific types of cards are explained to only be playable during X.

mob cards can only be played during X phase, instant cards can be played anytime, Y cards can be played when their conditions are met.